be/src/runtime/fragment_mgr.cpp

```diff
@@ -753,12 +753,12 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params,
         {
             SCOPED_RAW_TIMER(&duration_ns);
             auto prepare_st = context->prepare(params);
-            g_fragmentmgr_prepare_latency << (duration_ns / 1000);
             if (!prepare_st.ok()) {
                 context->close_if_prepare_failed();
                 return prepare_st;
             }
         }
+        g_fragmentmgr_prepare_latency << (duration_ns / 1000);
```

Comment:
what does ```<<``` mean here? same as ```=```?

Reply:
No, it's append-like operation. You can have a look at the func `operator<<` in `https://github.com/apache/brpc/blob/master/src/bvar/latency_recorder.cpp`